docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -437,3 +445,38 @@ See [Configure Grafana]({{< relref "../../../configure-grafana#authazuread" >}})
+| `scopes`                     | No       | Yes                | List of comma- or space-separated OAuth2 scopes.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | `openid email profile` |
+| `allow_sign_up`              | No       | Yes                | Controls Grafana user creation through the Azure AD/Entra ID login. Only existing Grafana users can log in with generic OAuth if set to `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                           | `true`                 |
+| `auto_login`                 | No       | Yes                | Set to `true` to enable users to bypass the login screen and automatically log in. This setting is ignored if you configure multiple auth providers to use auto-login.                                                                                                                                                                                                                                                                                                                                                                                                                                     | `false`                |
+| `role_attribute_path`        | No       | Yes                | [JMESPath](http://jmespath.org/examples.html) expression to use for Grafana role lookup. Grafana will first evaluate the expression using the OAuth2 ID token. If no role is found, the expression will be evaluated using the user information obtained from the UserInfo endpoint. The result of the evaluation should be a valid Grafana role (`None`, `Viewer`, `Editor`, `Admin` or `GrafanaAdmin`). For more information on user role mapping, refer to [Configure role mapping]({{< relref "#configure-role-mapping" >}}).                                                                          |                        |
```

Comment:
I see `#map-roles` header in the page, but not `#configure-role-mapping`.

Reply:
Done